src/main/java/spireCafe/util/decorationSystem/DecorationSystem.java

```diff
@@ -0,0 +1,166 @@
+    }
+
+    private void initWindows() {
+        float x1 = 490f * Settings.scale;
+        float x2 = 1430f * Settings.scale;
+        float maxWiggle = 50f * Settings.scale;
+
+        // Figure out which prefix to use based on dark vs. light background
+        boolean darkBg = ((CafeRoom) AbstractDungeon.getCurrRoom().event).darkBg;
+        String noLights  = darkBg ? "window_dark_noLights"  : "window_light_noLights";
+        String lights    = darkBg ? "window_dark_lights"    : "window_light_lights";
+
+        // Randomly decide which one goes first vs. second
+        boolean firstIsNoLights = rng.randomBoolean();
+        String firstWindow  = firstIsNoLights ? noLights : lights;
+        String secondWindow = firstIsNoLights ? lights   : noLights;
+
+        // Always place the first window
+        WindowDecoration w = new WindowDecoration(firstWindow);
+        w.move(x1 + rng.random(-maxWiggle, maxWiggle), w.fixedY);
+        decorations.add(w);
+
+        // Optionally place the second window
+        if (rng.randomBoolean()) {
+            w = new WindowDecoration(secondWindow);
+            w.move(x2 + rng.random(-maxWiggle, maxWiggle), w.fixedY);
+            decorations.add(w);
+        }
+    }
+
+    public void render(SpriteBatch sb) {
+        sb.setColor(Color.WHITE);
+        decorations.forEach(d -> d.render(sb));
+    }
+
+    public void update() {
+        if (Settings.isDebug && !AbstractDungeon.isScreenUp && InputHelper.justClickedRight && CardCrawlGame.playerName.equals("rorDev")) {
+            decorations.clear();
+            spawnDecorations();
+        }
+
+        decorations.forEach(Decoration::update);
+    }
+
+    public void dispose() {
+        decorations.clear();
+    }
+
+    private static void initAllDecorations() {
+        allDecorations = new ArrayList<>();
+
+        allDecorations.add(new Decoration("bold_and_brash", TexLoader.getTexture(Anniv7Mod.makeUIPath("decoration/bold_and_brash.png")), 0, 0));
+        allDecorations.add(new Decoration("cultist_portrait", TexLoader.getTexture(Anniv7Mod.makeUIPath("decoration/cultist_portrait.png")), 0, 0));
+        allDecorations.add(new Decoration("framed_louse", TexLoader.getTexture(Anniv7Mod.makeUIPath("decoration/framed_louse.png")), 0, 0));
+        allDecorations.add(new Decoration("poster", TexLoader.getTexture(Anniv7Mod.makeUIPath("decoration/poster.png")), 0, 0));
+        allDecorations.add(new Decoration("potted_plant", TexLoader.getTexture(Anniv7Mod.makeUIPath("decoration/potted_plant.png")), 0, 0));
+        allDecorations.add(new Decoration("spire_cafe_sign", TexLoader.getTexture(Anniv7Mod.makeUIPath("decoration/spire_cafe_sign.png")), 0, 0));
+        allDecorations.add(new StackedShelfDecoration());
+
+        //allDecorations.add(new Decoration("shelf", TexLoader.getTexture(Anniv7Mod.makeUIPath("decoration/shelf.png")), 0, 0));
+
+        /*Could be used for bar decoration
```

Comment:
To avoid having unused files around, should we delete these until we're ready to use them?